knowledge/craftable_spawneggs.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: page
+title: craftable_spawners
```

Comment:
Title is used to create the side-bar. URL is built from the actual filename.

Lets change title to: Craftable Spawn-Eggs